Add missing meta descriptions (#488)
mongodb/docs-golang: source/fundamentals/enterprise-auth.txt

.. meta::
   :keywords: security, code example, credentials

>    :description: Learn how to authenticate in MongoDB using Enterprise Edition mechanisms like GSSAPI/Kerberos and LDAP, with examples for Go driver integration.

Reviewer: ```suggestion
   :description: Learn how to authenticate with the MongoDB Go Driver using Enterprise Edition mechanisms like GSSAPI/Kerberos and LDAP, with examples.
```
